### PR TITLE
Remove development-environment references

### PR DESCRIPTION
**Pull request by ErikSchierboom**
As I am no longer using the development-environment setup, keeping that working an up-to-date is becoming more of a chore.
We also don't accept PRs to the website, so we don't _need_ to support a version of the website that runs via Docker.
I'll be submitting more PRs to other repos to clean them up.﻿

### CODE REVIEW

**Review by kntsoriano — APPROVED** [review]

**Comment by iHiD** [comment]
Any reason this is still a draft?

**Comment by iHiD** [comment]
@ErikSchierboom I've pushed some changes to the README. Let me know what you think.

File: README.md [hunk 1]
```diff
@@ -4,83 +4,49 @@
 [![Maintainability](https://api.codeclimate.com/v1/badges/b47ec4d5081d8abb59fa/maintainability)](https://codeclimate.com/github/exercism/website/maintainability)
 [![View performance data on Skylight](https://badges.skylight.io/typical/VNpB7GqXZDpQ.svg)](https://oss.skylight.io/app/applications/VNpB7GqXZDpQ)
 
-This is the website component of Exercism. It is Ruby on Rails app, backed by MySQL. It also relies on Redis and AnyCable.
+This is the website component of Exercism.
+It is Ruby on Rails app, backed by various services.
 
-## Running within our development-environment
+_Note: This repo is treated as an internal repo. That means the source code is public, but we do not accept Pull Requests and we do not officially support the app being run locally._
```
Comment (iHiD) [inline README.md]:
```suggestion
**Please note: This repo is treated as an internal repo. That means the source code is public, but we do not accept Pull Requests and we do not officially support the app being run locally.**
```

**Review by ErikSchierboom — COMMENTED** [review]
Changes look good to me!

**Review by iHiD — APPROVED** [review]
Nice work :)

**Pull request merged**